manifests/o/OpenSC/OpenSC/0.23.0/OpenSC.OpenSC.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: OpenSC.OpenSC
+PackageVersion: 0.23.0
+PackageLocale: en-US
+Publisher: OpenSC
+PublisherUrl: https://github.com/OpenSC
+PublisherSupportUrl: https://github.com/OpenSC/OpenSC/issues
+PackageName: OpenSC

```

```suggestion
PackageName: OpenSC smartcard framework
```
To match Apps&Features / Control Panel